lightly/models/modules/heads.py

```diff
@@ -370,14 +370,20 @@ class SwaVPrototypes(ProjectionHead):
 
     """
     def __init__(self,
-                 input_dim: int = 128,
-                 n_prototypes: int = 3000):
-        super(SwaVPrototypes, self).__init__([])
-        self.layers = nn.Linear(input_dim, n_prototypes, bias=False)
-
-    def normalize(self):
-        """Normalizes the prototypes so that they are on the unit sphere."""
-        utils.normalize_weight(self.layers.weight)
```

Comment:
I don't think this should be deleted. Instead we should loop over all heads and normalise them.

Reply:
I think this is the only thing that is missing then we should be good to merge :)